### ISSUE TITLE
npm ERR! code ERR_SSL_CIPHER_OPERATION_FAILED

### ISSUE THREAD

**Issue by lakeofthewoods1** [hashlips-lab/nft-erc721-collection#195]
I am getting this error, I tried different versions of node and I couldn´t fix it. It is frustrating as until here everything worked fine. I am just a hair from beeing able to upload my NFTs. May someone help me?

npm ERR! code ERR_SSL_CIPHER_OPERATION_FAILED
npm ERR! D46A0000:error:1C800066:Provider routines:ossl_gcm_stream_update:cipher operation failed:c:\ws\deps\openssl\openssl\providers\implementations\ciphers\ciphercommon_gcm.c:312:
npm ERR!

**Comment by liarco** [comment]
Hi @lakeofthewoods1,
I'm pretty sure this has nothing to do with this project (you. might have some configuration issue on your machine), but you didn't even tell us which command you are running... please give us more info so we can try to understand what's happening. Thank you.

Also, please make sure you followed all the steps in my video about how to install all dependencies on any operating system: https://github.com/hashlips-lab/nft-erc721-collection#youtube-tutorials

**Comment by lakeofthewoods1** [comment]
Hello, thank you so much for your answer. I was able to install some dependencies that I wasn't aware of. Now I am at the point of verifying the contract with the command yarn verify and I get this error:

Successfully submitted source code for contract
contracts/MonarchGlider.sol:MonarchGlider at 0xd13EDB2D951a55EAC161cC7aa400acb91e7C73b0
for verification on the block explorer. Waiting for verification result...

We tried verifying your contract MonarchGlider without including any unrelated one, but it failed.
Trying again with the full solc input used to compile and deploy it.
This means that unrelated contracts may be displayed on Etherscan...

Successfully submitted source code for contract
contracts/MonarchGlider.sol:MonarchGlider at 0xd13EDB2D951a55EAC161cC7aa400acb91e7C73b0
for verification on the block explorer. Waiting for verification result...

Error in plugin @nomiclabs/hardhat-etherscan: The contract verification failed.
Reason: Fail - Unable to verify

For more info run Hardhat with --show-stack-traces
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
PS D:\Downloads D\nft-erc721-collection-2.0.0\nft-erc721-collection-2.0.0\smart-contract>

I tried reading the links but I am not able to move on.

Thank you
